### ISSUE TITLE
CSGO Update 2023.02.03 broke the Gamedata

### ISSUE THREAD

**Issue by SaengerItsWar** [hermansimensen/eventqueue-fix#22]
Hey,
the new CSGO Update broke the AddEventThree signature.

```L 02/03/2023 - 22:58:47: [SM] Exception reported: Faild to find AddEventThree signature.
L 02/03/2023 - 22:58:47: [SM] Blaming: eventqueuefix.smx
L 02/03/2023 - 22:58:47: [SM] Call stack trace:
L 02/03/2023 - 22:58:47: [SM]   [0] SetFailState
L 02/03/2023 - 22:58:47: [SM]   [1] Line 152, eventqueuefix.sp::LoadDHooks
L 02/03/2023 - 22:58:47: [SM]   [2] Line 50, eventqueuefix.sp::OnPluginStart
L 02/03/2023 - 22:58:47: [SM] Unable to load plugin "eventqueuefix.smx": Error detected in plugin startup (see error logs)
```

**Comment by rtldg** [comment]
Can you test if this works and doesn't break anything https://github.com/rtldg/eventqueue-fix/commit/d91a9745897160a29c74c8da138f874288063f23

**Comment by pedrotski** [comment]
> Can you test if this works and doesn't break anything [rtldg@d91a974](https://github.com/rtldg/eventqueue-fix/commit/d91a9745897160a29c74c8da138f874288063f23)

Seems to load and play fine. That's all the information I can provide.

**Comment by rtldg** [comment]
I updated it a bit further to fix the AddEventThree signature... again lmao
https://github.com/hermansimensen/eventqueue-fix/compare/main...rtldg:eventqueue-fix:csgolinux-gd-up
I'm going to make a PR for it but I haven't personally tested other than asking other people to test for me
